cmake/version.cmake

```diff
@@ -16,6 +16,9 @@ set(FLAMEGPU_VERSION_PRERELEASE "${CMAKE_MATCH_1}")
 
 # @todo - validate that these have been found correctly via regex? Although it should get caught by the below validation, but errors could be better.
 
+# Generate a unique build hash
+string(RANDOM LENGTH 36 FLAMEGPU_BUILD_HASH)
+
```

Comment:
Can you explain the utility of this value?

If people build FGPU themselves, rather than using a prebuilt binary they will have a different value.
Surely commit hash (which I think is included in one of the other version strings) has more utility.

Reply:
If you want to try and uniquely identify users, you need to hash a bunch of machine info.

Reply:
It's also not a hash, if its just a random number. The name isn't good.